lib/components/form/connect-location-field.js

```diff
@@ -80,9 +80,19 @@ export default function connectLocationField(
     }
 
     const geocoderConfig = config.geocoder
-    if (currentPosition?.coords) {
-      const { latitude: lat, longitude: lon } = currentPosition.coords
-      geocoderConfig.focusPoint = { lat, lon }
+    if (geocoderConfig) {
+      if (query.to && query.to?.lat && query.to?.lon) {
+        const { lat, lon } = query.to
+        geocoderConfig.focusPoint = { lat, lon }
+      }
+      if (query.from && query.from?.lat && query.from?.lon) {
+        const { lat, lon } = query.from
+        geocoderConfig.focusPoint = { lat, lon }
+      }
+      if (currentPosition?.coords) {
```

Comment:
Should these be `else if` blocks? Right now, the order of priority reads like:
1. current position
2. from
3. to

Reply:
this is the intended priority order